### ISSUE TITLE
Tensorflow Model Pruning 

### ISSUE THREAD

**Issue by SEETHAMRAJU** [microsoft/nni#2527]
I have been trying to prune a tensorflow keras model using the nni package. However after pruing i am trying to check the flops but htere is no improvement in the flops. Also the model weights are of the same size. I followed the tensorflow fpgm pruner example. Please help me with this. Is there any speedup api for tensorflow.

thank you

**Comment by liuzhe-lz** [comment]
Current TensorFlow model compression module was designed for TF v1.x and is no longer maintained. You may try out PyTorch version for now.
And by default NNI model compression only mask the weigths to simulate accuracy. You will need [speed up module](https://github.com/microsoft/nni/blob/master/docs/en_US/Compressor/ModelSpeedup.md) to accelerate inference.

**Comment by SEETHAMRAJU** [comment]
Thanks for the information **liuzhe-lz**. For model speedup in torch I am getting 
_cannot import name 'ModelSpeedup'_. Is it some probelm regrading the version of nni.
Thankyou,

**Comment by liuzhe-lz** [comment]
The package structure was updated in version v1.6.
You can try `from nni.compression.speedup.torch import ModelSpeedup` if you are using v1.5.

**Comment by yangxudong9** [comment]
> The package structure was updated in version v1.6.
> You can try `from nni.compression.speedup.torch import ModelSpeedup` if you are using v1.5.

That's worked. Thanks for your reply  @liuzhe-lz. But my version is v1.6.Should I install NNI from source code?
`
>>> import nni
>>> nni.__version__
'1.6'
`

**Comment by liuzhe-lz** [comment]
Oh, seems the PR (#2501) was merged after v1.6 release. It only changed package structure and did not affect functionality.

**Comment by scarlett2018** [comment]
> > The package structure was updated in version v1.6.
> > You can try `from nni.compression.speedup.torch import ModelSpeedup` if you are using v1.5.
> 
> That's worked. Thanks for your reply @liuzhe-lz. But my version is v1.6.Should I install NNI from source code?
> `
> 
> > > > import nni
> > > > nni.**version**
> > > > '1.6'
> > > > `

closing the issue as it has been resolved.

**Comment by Nhiem** [comment]
Hello Everyone because it's a bit difficult for me to find the right place to ask the question... My question related to the pruning parameter. Im a bit confusing in the model parameter as the model I created as shown in Figure 1 with 5.188.904 number of parameter; However after I prune the model with sparsity 60% for all Conv2D layers then my model parameter increasing to figure 2.. with 7.314.527 number of parameter.. Anyone could help Im really appreciate for your time 

Figure 1 model number of parameters first original created 
![image](https://user-images.githubusercontent.com/45351965/99079630-2acc8e80-25fb-11eb-8fc0-008f9a7dd77d.png)

Figure 2 model number of total parameter after pruning 60%
![image](https://user-images.githubusercontent.com/45351965/99079938-9282d980-25fb-11eb-91e2-c2b02e728ae7.png)
